content/en/md_conversion_test_files/common_components/plain_markdown.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Plain Markdown Elements
+---
+
+## Expected .md output
+
+- Most of this should remains unchanged, or uses a format that does not meaningfully differ from its original form. 
+  - Some spacing changes can be difficult to avoid.
+  - OOTB Markdoc does not support description lists, but a similar structure can be used. (Longer term, Markdoc can be patched to support it, as we've already done with Cdocs, but it introduces complexity that we probably don't want to deal with at this stage.)
+- In some cases, content may be slightly optimized for agents (for example, links will become inline).
+
+## Headings
+
+### Heading level 3
+
+Some text.
+
+#### Heading level 4 {#heading-level-4}
+
+Some text, with a custom heading ID.
+
+##### Heading level 5
+
+Some text.
+
+###### Heading level 6
+
+Some text.
+
+## Inline text formatting
+
+You can *italicize* text, or **bold** it.
```

Comment:
Do we need to account for using underscores to italicize and bold? This line could also be sourced as 
`You can _italicize_ text, or __bold__ it.`

Reply:
I believe Markdoc will pick it up either way, but I've added examples to make sure those instances are processed correctly, thank you!